vm/message/src/unsigned_message.rs

```diff
@@ -164,6 +165,34 @@ impl Message for UnsignedMessage {
         let total: TokenAmount = self.gas_fee_cap() * self.gas_limit();
         total + self.value()
     }
+    fn valid_for_block_inclusion(&self, min_gas: i64) -> Result<(), String> {
+        if self.version != 0 {
+            return Err(format!("Message version: {} not  supported", self.version));
+        }
+        if self.value < 0.into() {
```

Comment:
```suggestion
        if self.value.sign() == Sign::Minus {
```

Is better than constructing a bigint to compare with